build.gradle

```diff
@@ -0,0 +1,48 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+}
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '2.0.4'
+    id 'java-library'
+}
+
+//This allows you to build a single shadowJar with the contents of all the
+// subprojects included in it
+dependencies {
+    compile subprojects
+}
+
+subprojects {
+    apply plugin: 'java-library'
+
+    group 'org.htsjdk'
+    version '0.0.1'
+    sourceCompatibility = 1.8
```

Comment:
Should we also add explicitly `targetCompatibility` (https://docs.gradle.org/current/userguide/java_plugin.html#other_convention_properties), to be able to change them independently (related with my first comment in this review)?

Reply:
yes, good idae.